salesforce/modules/soap/tests/test.bal

```diff
@@ -75,7 +75,7 @@ function testconvertLead() {
 
 @test:AfterSuite {}
 function testDeleteRecord() returns error? {
-    check restClient->deleteAccount(accountId);
-    check restClient->deleteContact(contactId);
-    check restClient->deleteOpportunity(opportunityId);
+    check restClient->delete("Account", accountId);
+    // check restClient->delete("Contact", contactId);
+    // check restClient->delete("Opportunity", opportunityId);
```

Comment:
If unnecessary let's remove them or let's enable them if necesssary

Reply:
Removed in 8bfb122ccf270a0868c40a7ea6ddaef1d3a79a39